docs/changelog.txt

```diff
@@ -42,6 +42,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 ## Misc Improvements
 
 ## Documentation
+- Added `modding guide <modding-guide>`
```

Comment:
Just
```
- Added `modding-guide`
```
is sufficient. When it's rendered, it will expand to a link with anchor text equal to the title of the document. E.g. it will show as:
`- Added DFHack modding guide`

Reply:
Ah, I assumed it would be the label text. Cool and good.